nano/node/node.cpp

```diff
@@ -23,6 +23,7 @@ std::chrono::seconds constexpr nano::node::search_pending_interval;
 int constexpr nano::port_mapping::mapping_timeout;
 int constexpr nano::port_mapping::check_timeout;
 unsigned constexpr nano::active_transactions::announce_interval_ms;
+unsigned constexpr nano::active_transactions::confirmation_request_delay_ms;
```

Comment:
Would be nice to use chrono units here to avoid use-site casts (like with arrival_time_min), but since this is consistent with announce_interval_ms etc, maybe we can do a separate cleanup patch for all uses.

Reply:
Good point!